rskj-core/src/test/java/co/rsk/peg/BridgeUtilsLegacyTest.java

```diff
@@ -3,16 +3,22 @@
 import co.rsk.bitcoinj.core.Address;
 import co.rsk.bitcoinj.core.BtcTransaction;
 import co.rsk.bitcoinj.core.Coin;
+import co.rsk.bitcoinj.core.UTXO;
 import co.rsk.config.BridgeConstants;
 import co.rsk.config.BridgeMainNetConstants;
 import co.rsk.config.BridgeRegTestConstants;
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
We try to stay away from external dependencies as much as possible. See https://github.com/rsksmart/rskj/issues/806 

Perhaps `testGetAmountSentToAddress` could simply receive the address as a param. Let me know if you wanna discuss other possible solutions

Reply:
Of course I want. What about creating our own custom pair container?

Reply:
Done